ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.9__4.5.yaml

```diff
@@ -40,25 +37,39 @@ tag_specification:
   name: "4.5"
   namespace: ocp
 tests:
-- as: e2e-aws-ocp-45
+- as: operator-e2e-aws-ocp-45
   steps:
+    allow_skip_on_success: true
     cluster_profile: aws
     test:
-    - as: serverless-e2e
+    - as: operator-e2e
```

Comment:
should I name that `serverless-e2`? or `operator-...` ?

Reply:
I suppose it could be `serverless-e2e` like in the other blocks. The important name is `- as: operator-e2e-aws-ocp-45` which is shown on pull requests to differentiate the checks.